internal/context/init.go

```diff
@@ -38,31 +38,55 @@ func Init(projectName string, outDir string) *projectconfig.ZeroProjectConfig {
 	projectConfig.Context["ShouldPushRepoUpstream"] = promptPushRepoUpstream()
 	projectConfig.Context["GithubRootOrg"] = promptGithubRootOrg()
 	projectConfig.Context["githubPersonalToken"] = promptGithubPersonalToken(projectName)
-	projectConfig.Modules = chooseStack(getRegistry())
 
 	// chooseCloudProvider(&projectConfig)
 	// fmt.Println(&projectConfig)
 	// s := project.GetSecrets(rootDir)
 	// fillProviderDetails(&projectConfig, s)
 	// fmt.Println(&projectConfig)
+	moduleSources := chooseStack(getRegistry())
+	moduleConfig := loadAllModules(moduleSources)
+	for _ = range moduleConfig {
+		// TODO: initialize module structs inside project
+	}
+
+	projectParameters := promptAllModules(moduleConfig)
+	for _ = range projectParameters {
```

Comment:
do we want to temporarily add projectParams back into the context , so it can preserve backwards compatibility before we have the `modules` struct 
```suggestion

	for k, v = range projectParameters {
	  projectConfig.Context[k] = v
```